pandas/io/parsers.py

```diff
@@ -2135,6 +2148,53 @@ def _maybe_parse_dates(self, values, index, try_parse_dates=True):
         return values
 
 
+class ArrowParserWrapper(ParserBase):
+    """
+
+    """
```

Comment:
Doc-string of any kind?

Reply:
this was done based on the CParserWrapper which has a similar empty docstring. should i remove it?

Reply:
Given that this is a slightly unusual one, I would briefly describe it.